Billwilke42/ideabox-boilerplate
# Spike/refactor

**What's this PR do?**
- Adjusts HTML and CSS to reflect proper indentation according to their respective style guides.
- Groups similar CSS classes to share repetitive styles
- Adds styling for "Search Image" and brings it in line with the "Search Input"
- Moves assets out of unnecessary additional folder which was causing issues with 'search image styling'

**Where should the reviewer start?**
- Changes in HTML begin within the 'main' element
- Changes in CSS begin between lines 41 and 90
- Review changes to icon assets

**How should this be manually tested?**
- Open index.html in browser and check differences against previous master branches index.html
